seedlet/inc/wpcom-customize-preview.js

```diff
@@ -18,4 +17,74 @@
 			}
 		} );
 	} );
+
+	// The wpcom colors plugin does not handle live updates of extra CSS.
+	// This function provides the CSS updates via JavaScript.
+	wp.customize( 'colors_manager[colors]', function( value ) {
+		value.bind( function( to ) {
+			const { bg, fg1, fg2 } = to;
+			const extraCSS = `/*
```

Comment:
It's unfortunate that some version of this CSS has to live in three places (here, `inc/wpcom-colors.php`, and `inc/wpcom-editor-colors.php`), but I'm not sure how to get around this if we want to support custom colors as the plugin is written right now.

Reply:
Agreed — it's less than ideal. I think this might be the sort of thing we implement for now, and then re-evaluate later on. 

For `inc/wpcom-colors.php`, and `inc/wpcom-editor-colors.php`, could we write a single function for it in `wpcom.php`, and then call it in each of those files? We could use some sort of context parameter to fill in the appropriate classnames.

Reply:
The extra css is added via `add_theme_support` here: https://github.com/Automattic/themes/blob/try/seedlet/color-annotations/seedlet/inc/wpcom-colors.php#L517

Can this function be called passing an argument for the context?

Reply:
I'm not really sure, to be honest. 😄 `add_theme_support` [can generally take extra arguments](https://developer.wordpress.org/reference/functions/add_theme_support/), but it looks like the function name here _is_ the extra argument.

Reply:
Yeah not totally sure what to do here as the argument in this case refers to callback's handle. I don't think this optimization should be a blocker though.

I'll add some comments to clarify what's going on, because, as we spoke about, these functions are rather opaque.

Reply:
Yeah, I wish we didn’t have to put this extra CSS in here, but it does work well and child themes won’t have to touch it either which is a plus. 

I agree we should keep it and iterate on it later :-)